movo_common/movo_ros/src/movo_jtas/movo_jtas.py

```diff
@@ -308,6 +313,8 @@ def now_from_start(start):
         # Continue commanding goal until success or timeout
         self._last_movement_time = rospy.get_time()
         self._last_gripper_pos = self._gripper_fdbk.position
+        # mapping 0 = fully closed, 56 = fully open
```

Comment:
why do you have to change the gripper logic?
Are you sure you were using it right in the first place? have u looked at how the rest of the kinova code uses the gripper? Opening and closing the gripper works without these changes (as can be seen in the init sequence). Maybe u were just commanding the gripper incorrectly?
How does this change affect the init sequence

Reply:
I realized this should not be done, and there is a deg->rad conversion missing somewhere in kinova's code. movo_jtas expects the incomming value to be in radians, and moveit uses radians so from our side everything will work. the mismatch is actually in rviz which uses degrees, I also believe the urdf might be the spot that is missing the deg->rad conversion